src/private/remoteCamera.ts

```diff
@@ -134,11 +136,6 @@ export namespace remoteCamera {
     ControlDisabled,
   }
 
-  GlobalVars.handlers['remoteCamera.capableParticipantsChange'] = handleCapableParticipantsChange;
-  GlobalVars.handlers['remoteCamera.handlerError'] = handleHandlerError;
```

Comment:
How are we handling these registration ?

Reply:
For all handlers that don't have a default / always needs to run behavior (does nothing if there is no handlers), the registration is done in the "registerXXXHandler" call. The rest will use initialize to register the default handler and use a handler function variable to register the handler.